src/browser/provider/built-in/phantomjs.js

```diff
@@ -0,0 +1,47 @@
+import { create as createPhantom } from 'phantom';
+
+export default {
+    phantom: null,
+
+    openedPages: {},
```

Comment:
Also, `listAvailableOptionalBrowserNames` should throw if not implemented by default. And we should perform check of `hasOptionalBrowserNames` flag before using it